[Bug]: tests not running on new otter webapp

### Package name

testing

### Package version

10.1.1

### Reproduction steps

```shell
npm create @o3r otter-project
cd otter-project
ng g application webapp
# choose preset BASIC, Jest and Playwright when prompted
npm run test -w webapp
```

### Current result

```shell
$ npm run test -w webapp

> webapp@0.0.0-placeholder test
> jest

Error: Cannot find module '../../tsconfig.base'
Require stack:
- /Users/vscaiceanu/workspace/temp/quickstart-otter/jest.config.ut.js
- /Users/vscaiceanu/workspace/temp/quickstart-otter/apps/webapp/jest.config.js
...
```

### Expected result

```shell
$ npm run test -w webapp

> webapp@0.0.0-placeholder test
> jest

 PASS   webapp  src/app/app.component.spec.ts
  AppComponent
    ✓ should create the app (51 ms)
    ✓ should have the 'webapp' title (6 ms)
    ✓ should render title (10 ms)

Test Suites: 1 passed, 1 total
Tests:       3 passed, 3 total
Snapshots:   0 total
Time:        2.395 s
Ran all test suites.
```

### Additional comments

_No response_

--- a/packages/@o3r/testing/schematics/ng-add/playwright/index.ts
+++ b/packages/@o3r/testing/schematics/ng-add/playwright/index.ts
@@ -44,6 +44,7 @@ export function updatePlaywright(options: NgAddPackageOptions, dependencies: Rec
 # Playwright
 dist-e2e-playwright
 playwright-reports
+test-results
 `;
         tree.overwrite(gitignorePath, gitignore);
       }

--- a/packages/@o3r/testing/schematics/ng-add/playwright/templates/e2e-playwright/scenarios/__name@dasherize__.e2e-playwright-spec.ts
+++ b/packages/@o3r/testing/schematics/ng-add/playwright/templates/e2e-playwright/scenarios/__name@dasherize__.e2e-playwright-spec.ts
@@ -6,7 +6,7 @@ export class <%= classify(scenarioName) %> extends BaseScenario {
     test.describe.serial('Empty <%= classify(scenarioName) %> tests', () => {
       test('Empty test', async ({ page }) => {
         await page.goto(this.targetUrl);
-        await expect(page).toHaveTitle('TestApp');
+        await expect(page.locator('body')).toBeAttached();
       });
     });
   }

--- a/packages/@o3r/testing/schematics/playwright/scenario/templates/__name@dasherize__.e2e-playwright-spec.ts
+++ b/packages/@o3r/testing/schematics/playwright/scenario/templates/__name@dasherize__.e2e-playwright-spec.ts
@@ -6,7 +6,7 @@ export class <%= classify(scenarioName) %> extends BaseScenario {
     test.describe.serial('Empty <%= classify(scenarioName) %> tests', () => {
       test('Empty test', async ({ page }) => {
         await page.goto(this.targetUrl);
-        await expect(page).toHaveTitle('TestApp');
+        await expect(page.locator('body')).toBeAttached();
       });
     });
   }

